src/pages/detail-page/subpages/tab-panels/MetadataInformationPanel.tsx

```diff
@@ -1,7 +1,48 @@
-import React from "react";
+import React, { useEffect, useMemo, useState } from "react";
+import { useDetailPageContext } from "../../context/detail-page-context";
+import NavigatablePanel from "../components/NavigatablePanel";
+import ContactBlock from "../components/ContactBlock";
 
 const MetadataInformationPanel = () => {
-  return <div>MetadataInformationCard</div>;
+  const context = useDetailPageContext();
+  const metadataContact = context.collection
+    ?.getContacts()
+    ?.filter((contact) => contact.roles.includes("metadata"));
```

Comment:
not a too big deal, but consider using useMemo

```javascript
const metadataContacts = useMemo(() => {
  return context.collection?.getContacts()?.filter((contact) => contact.roles.includes("metadata")) || [];
}, [context.collection]);
```

Reply:
for performance

Reply:
Thank you for reminding. It should use useMemo. I forgot it.